tests/expectations/metrics/test_metrics_util.py

```diff
@@ -127,3 +155,133 @@ def test_sql_statement_conversion_to_string_bigquery(test_backends):
+        unexpected_index_column_names=unexpected_index_column_names,
+    )
+    assert unexpected_index_list == unexpected_index_list_two_index_columns
+
+
+@pytest.mark.unit
+def test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpected_index_columns_one_column(
+    pandas_animals_dataframe_for_unexpected_rows_and_index,
+    unexpected_index_list_one_index_column,
+):
+    dataframe: pd.DataFrame = pandas_animals_dataframe_for_unexpected_rows_and_index
+    updated_dataframe: pd.DataFrame = dataframe.set_index(["pk_1", "pk_2"])
+    expectation_domain_column_name = "animals"
+    unexpected_index_column_names = ["pk_1"]
+
+    unexpected_index_list = get_unexpected_indices_for_multiple_pandas_named_indices(
+        df=updated_dataframe,
+        expectation_domain_column_name=expectation_domain_column_name,
+        unexpected_index_column_names=unexpected_index_column_names,
+    )
+    assert unexpected_index_list == unexpected_index_list_one_index_column
+
+
+@pytest.mark.unit
+def test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpected_index_columns_wrong_column(
+    pandas_animals_dataframe_for_unexpected_rows_and_index,
+):
+    dataframe: pd.DataFrame = pandas_animals_dataframe_for_unexpected_rows_and_index
+    updated_dataframe: pd.DataFrame = dataframe.set_index(["pk_1", "pk_2"])
+    expectation_domain_column_name = "animals"
+    unexpected_index_column_names = ["i_dont_exist"]
+    with pytest.raises(MetricResolutionError) as e:
+        get_unexpected_indices_for_multiple_pandas_named_indices(
+            df=updated_dataframe,
+            expectation_domain_column_name=expectation_domain_column_name,
+            unexpected_index_column_names=unexpected_index_column_names,
+        )
+    assert e.value.message == (
+        "Error: The column i_dont_exist does not exist in the named indices. Please "
+        "check your configuration"
+    )
+
+
+@pytest.mark.unit
+def test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpected_index_wrong_domain(
+    pandas_animals_dataframe_for_unexpected_rows_and_index,
+):
+    dataframe: pd.DataFrame = pandas_animals_dataframe_for_unexpected_rows_and_index
+    updated_dataframe: pd.DataFrame = dataframe.set_index(["pk_1", "pk_2"])
+    expectation_domain_column_name = None
+    unexpected_index_column_names = ["pk_1"]
+    with pytest.raises(MetricResolutionError) as e:
+        get_unexpected_indices_for_multiple_pandas_named_indices(
+            df=updated_dataframe,
+            expectation_domain_column_name=expectation_domain_column_name,
+            unexpected_index_column_names=unexpected_index_column_names,
+        )
+    assert e.value.message == (
+        "Error: The domain column is currently set to None. Please check your "
+        "configuration."
+    )
+
+
+@pytest.mark.unit
+def test_get_unexpected_indices_for_multiple_pandas_named_indices(
```

Comment:
nit: Could you move this tests so it is next to `test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpected_index_columns` since it's basically the same except for how the index column names are defined.

Reply:
Moving to other location. Thank you for the suggestion :)

Reply:
Done :)